tools/libs/light/libxl_pcid.c

```diff
@@ -148,7 +148,7 @@ static int pciback_dev_is_assigned(libxl__gc *gc, unsigned int domain,
     if (rc < 0 && errno == ENOENT)
         return 0;
     LOGE(ERROR, "Accessing %s", spath);
-    return -1;
+    return 0;
```

Comment:
Why this change is needed?

Reply:
This line can be executed only on development stage (perhaps because of selinux or something like this). It should not happen on product. Because of this I didn't implement careful error checking on caller side. So return -1 will be false-positive result on caller side.